content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -100,3 +100,55 @@ value. To recalculate the taxes, click :guilabel:`Compute taxes using Avatax`.
 
 .. tip::
    Tax calculation can even be done on negative subtotals and credit notes.
+
+Logging
+=======
+
+It is possible to log Avalara/*AvaTax* actions in Odoo for further analysis or verification of
+functionality. Logging is accessible through the *AvaTax* settings.
+
+To start logging *AvaTax* actions, first, navigate to the :menuselection:`Accounting app -->
+Configuration --> Settings`. Then, in the :guilabel:`Taxes` section, under the :guilabel:`AvaTax`
+settings, click :guilabel:`Start logging for 30 minutes`. Upon starting the log process, Odoo will
+log all Avalara/*AvaTax* actions performed in the database.
+
+To view the logs, click on :guilabel:`Show logs` to the right of the :guilabel:`Start logging for 30
+minutes`. This reveals a detailed list of Avalara/*AvaTax* actions. This list is sortable by the
+following columns:
+
+- :guilabel:`Created on` : this is a timestamp of the AvaTax calculation.
+- :guilabel:`Created by` : this is a numeric value of the user in the database.
+- :guilabel:`Database name` : this is the database name.
+- :guilabel:`Type` : there are two values this field can be, :guilabel:`Server` or
+  :guilabel:`Client`.
+- :guilabel:`Name` : this is the Avalara service name, in this case it will be *AvaTax*.
+- :guilabel:`Level` : by default this will be `INFO`.
+- :guilabel:`Path` : this indicates the path taken to make the calculation.
+- :guilabel:`Line` : this indicates the line that the calculation is made on.
+- :guilabel:`Function` : this indicates the calculation taken on the line.
```

Comment:
```suggestion
- :guilabel:`Created on`: timestamp of the *AvaTax* calculation.
- :guilabel:`Created by`: numeric value of the user in the database.
- :guilabel:`Database name`: name of the database.
- :guilabel:`Type`: two values can be chosen for this field, :guilabel:`Server` or
  :guilabel:`Client`.
- :guilabel:`Name`: Avalara service name. In this case, it will be *AvaTax*.
- :guilabel:`Level`: by default, this will be `INFO`.
- :guilabel:`Path`: indicates the path taken to make the calculation.
- :guilabel:`Line`: indicates the line the calculation is made on.
- :guilabel:`Function`: indicates the calculation taken on the line.
```